### PR TITLE
update presentation

### PR DESCRIPTION
**Pull request by lucadelu**

### CODE REVIEW

File: en/presentation/index.html [hunk 1]
```diff
@@ -568,6 +569,13 @@ <h3 style="color:#1166ad;">An Open Source Geospatial GNU/Linux Distribution</h3>
                      <p>Viking is a GPS data editor, analyser and viewer. It allows you to import and plot tracks and waypoints, show OpenStreetMap and Terraserver maps, download geocaches for a map area, make new tracks and waypoints, see your real time GPS position and more.</p>
                   </aside>
                </section>
+               <section>
```
Comment (kalxas) [inline en/presentation/index.html]: zyGrib is a desktop application... it should not be moved.

Comment (lucadelu) [inline en/presentation/index.html]: ok, but in osgeo live is in "maps and navigation" folder,

Comment (kalxas) [inline en/presentation/index.html]: your commit comment was mentioning move to Web Browser category...

Comment (lucadelu) [inline en/presentation/index.html]: Sorry I was wrong, I moved in "Navigation and Maps"

Comment (kalxas) [inline en/presentation/index.html]: ok thanks :)

**Comment by kalxas** [comment]
Closing this one, cleanup in
https://github.com/OSGeo/OSGeoLive-doc/pull/36

**Pull request closed without merging**